browsers/chrome-mv3/managed-schema.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Rather than have special cases for chrome-mv3 builds, I just added this small file from `browsers/chrome` to `browsers/chrome-mv3`.